src/basic/mount-util.c

```diff
@@ -943,3 +943,41 @@ int mount_option_mangle(
 
         return 0;
 }
+
+bool dev_is_devtmpfs(void) {
```

Comment:
not that it matters much, but I'd probably make this return "int" and then return "0" on "no devtmpfs" and "> 0" on "yes it is devtmpfs" and negative on error, when we can't determine it.

i.e. ignoring the error here is fine, but I think it's a better approach to do that in the caller rather than in the callee